.env.example

```diff
@@ -157,6 +160,10 @@
 ## Time each day at which to run the facet link group harvesting job. No default.
 # SCHEDULE_FACET_ENTRY_GROUPS_GENERATOR="05:30"
 
+## Times each day at which to run gallery validation, multiple values are
+# separated by spaces
+#SCHEDULE_GALLERY_VALIDATION="08:00 15:00"
```

Comment:
As well as the schedule and Rake task, do you think we should trigger the validation job immediately after each gallery is created or published? To run some of the more extensive checks like content type and download verification right away.

Reply:
yes, I checked this with Adrian and he thinks it would be useful